ts/components/CompositionArea.tsx

```diff
@@ -452,7 +452,7 @@ export function CompositionArea({
 
   const sendButtonFragment = (
     <>
-      <div className="CompositionArea__placeholder" />
+      <div hidden={!large} className="CompositionArea__placeholder" />
```

Comment:
It is a good practice to solve one problem per PR ( because if something goes wrong it can be reverted very easily ). Can you make another PR just for that?

Reply:
Yes, I will split this into another PR